pkg/datastore/filedb/filedb.go

```diff
@@ -149,6 +149,12 @@ func (f *FileDB) Find(ctx context.Context, col datastore.Collection, opts datast
 			return nil, err
 		}
 
+		// TODO: Remove this unnecessary log print.
+		f.logger.Info("filtering...",
+			zap.Any("entity", e),
+			zap.Any("filter", opts.Filters),
+		)
+
```

Comment:
this log print is just for debugging, lets me keep it as is and I will remove it after all find queries are ensured to work correctly 🙏

Reply:
nits, I think you should use Debug level.

Reply:
@knanao thanks for the comment 🙏  But let's me keep it as is, since it looks like the current logger only provides logs at info and error level.